source/extensions/extensions_metadata.yaml

```diff
@@ -532,6 +535,7 @@ envoy.filters.http.header_mutation:
   - envoy.filters.http.upstream
   security_posture: unknown
   status: alpha
+  status_upstream: alpha
```

Comment:
@wbpcode i think this where you need to chime in - i set to `alpha` for now as that is the general status - but i think we can now declare them separately

@alyssawilk just to confirm that is correct - something can be `alpha` for downstream and `stable` upstream ?

Reply:
oh if it's alpha for downstream lets leave it alpha for upstream too.
It's possible to be stable one way and not the other but in this case I don't think it's warranted.